### PR TITLE
feat(bulk-import): create bulk-import frontend plugin

### PR DESCRIPTION
**Pull request by debsmita1**
**Resolves:**
https://issues.redhat.com/browse/RHIDP-1399

**Screenshot:**
Screenshot 2024-02-27 at 12 22 48 AM

**_TODO_**
- Load the plugin as a dynamic one on the showcase

### CODE REVIEW

**Comment by ShiranHi** [comment]
Looks good to me!
I guess that Add button and the Actions columns will be added in a separate PR, right?

**Comment by debsmita1** [comment]
@jrichter1 ^

**Comment by debsmita1** [comment]
> Looks good to me! I guess that Add button and the Actions columns will be added in a separate PR, right?

Yes

File: plugins/bulk-import/tests/bulkImport.spec.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { expect, Page, test } from '@playwright/test';
+
+test.describe('Bulk import plugin', () => {
+  let page: Page;
+
+  test.beforeAll(async ({ browser }) => {
+    const context = await browser.newContext();
+    page = await context.newPage();
+    await page.goto('http://localhost:3000/bulk-import');
```
Comment (jrichter1) [inline plugins/bulk-import/tests/bulkImport.spec.ts:9]: We've added an [environment variable](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/quay/playwright.config.ts#L20) to override the base URL for the other plugins. Can you do that for the sake of consistency?

**Review by jrichter1 — APPROVED** [review]

**Comment by invincibleJai** [comment]
/approve


Looks good, yarn lock file needs an update.

**Review by gashcrumb — APPROVED** [review]

**Comment by invincibleJai** [comment]
Closed , as rebased with https://github.com/janus-idp/backstage-plugins/pull/1327

**Pull request closed without merging**